src/dotnet/MLSearch.Service/MLSearch.cs

```diff
@@ -1,9 +1,36 @@
 ﻿using ActualChat.Chat;
+using ActualChat.Users;
 
 namespace ActualChat.MLSearch;
 
-public class MLSearchImpl (ICommander commander): IMLSearch
+public class MLSearchImpl (ICommander commander, IMLSearchBackend backend, IChats chats, IAccounts accounts): IMLSearch
 {
+    private IChats Chats => chats;
+    private IAccounts Accounts => accounts;
+    private IMLSearchBackend Backend { get; } = backend;
+
+    private ICommander Commander { get; } = commander;
```

Comment:
It is totally fine to have such a private proxy property for primary constructor parameter. However, I personally decided not to use this pattern. The main reason behind that is both primary constructor parameter and proxy property are accessible for class members, so each time you want to use e.g. `ICommander` instance you have to select between `commander` and `Commander`. So, highly likely, over the time the code will access `ICommander` both ways bringing some degree of inconsistency.

Reply:
this property duplicates commander, can we just remove it?